dataplane/linux/int_dataplane.go

```diff
@@ -374,8 +374,8 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 	dp.ipSets = append(dp.ipSets, ipSetsV4)
 
 	if config.RulesConfig.VXLANEnabled {
-		routeTableVXLAN := routetable.New([]string{"vxlan.calico"}, 4, true, config.NetlinkTimeout, config.DeviceRouteSourceAddress,
-			config.DeviceRouteProtocol, true)
+		routeTableVXLAN := routetable.New([]string{"^vxlan.calico.*"}, 4, true, config.NetlinkTimeout,
```

Comment:
FWIW, I think there's only one VXLAN device so no need for a prefix.  @caseydavenport would "just know"